app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/ArticleSearchResultItem.vue

```diff
@@ -92,13 +74,16 @@ export default {
     },
   },
   methods: {
-    handleInsert() {
+    handleInsert(e) {
+      e.stopPropagation();
       this.$emit('insert', this.id);
     },
-    handlePreview() {
+    handlePreview(e) {
+      e.stopPropagation();
       this.$emit('preview', this.id);
     },
-    async handleCopy() {
+    async handleCopy(e) {
+      e.stopPropagation();
       await copyTextToClipboard(this.url);
```

Comment:
This throws an error. 
<img width="459" alt="Screenshot 2023-10-30 at 7 49 17 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/96f61244-7125-4239-bcea-9f6dc8b15fa9">